### DISCUSSION TITLE
Testing File Downloads

### DISCUSSION THREAD

**Discussion by srikanthgoudm** [getgauge/taiko#1986] [General]
I am looking for testing file downloads from applciation
is there any limitations for huge downloads

**Comment by sriv** [comment]
Taiko doesn't have a limit (at least to our knowledge). Are you facing any issue?

**Reply by srikanthgoudm** [reply]
how can we verify the downlods on remote machine

**Reply by sriv** [reply]
This example may help: https://github.com/getgauge/taiko/blob/master/examples/02-file_download.js
